packages/styleguide/src/components/App/App.js

```diff
@@ -263,4 +259,49 @@ const PageContent = styled.main`
   }
 `;
 
+const PageSidebar = styled(Sidebar)`
```

Comment:
Toto by som cele vybraz z App a dal do samotatneho componentu. Lebo zbytocne to predlzuje tento file a nie je to priamo spojene s App.